src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGeneratorOptions.cs

```diff
@@ -61,7 +62,19 @@ private bool DefaultDocInclusionPredicate(string documentName, ApiDescription ap
 
         private string DefaultOperationIdSelector(ApiDescription apiDescription)
         {
-            return apiDescription.ActionDescriptor.AttributeRouteInfo?.Name;
+            var actionDescriptor = apiDescription.ActionDescriptor;
+
+            // Resolve the operation ID from the route name and fallback to the
+            // endpoint name if no route name is available. This allows us to
+            // generate operation IDs for endpoints that are defined using
+            // minimal APIs.
+#if (!NETSTANDARD2_0)
```

Comment:
This is ifdef'd out since the M.A.Routing 2.1.0 package does not contain the `EndpointNameMetadata` types that we need here. We could update the repo to rely on M.A.* 2.2.0 packages for everything if we wanted to avoid if-defing here but that seemed more appropriate for a separate PR.